File: src/DailyJournalApplicationSolution/DailyJournalApplication/Models/Entry.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DailyJournalApplication.Models;

[Table("Entry")]
public partial class Entry
{
    [Key]
    [Column("ID")]
    public int Id { get; set; }

    [StringLength(64)]
    public string Date { get; set; }

    [StringLength(8)]
    public string Mood { get; set; }

    [StringLength(32)]
    public string Affirmation { get; set; }

    [StringLength(64)]
    public string Achievement { get; set; }

    [StringLength(64)]
    public string NonAchievement { get; set; }

    [StringLength(64)]
    public string TomorrowsGoal { get; set; }

    public int? Rating { get; set; }
}

File: src/DailyJournalApplicationSolution/DailyJournalApplication/ClientApp/src/entry.ts
export class Entry {
  date?: string | null;
  mood?: string | null;
  affirmation?: string | null;
  achievement?: string | null;
  nonAchievement?: string | null;
  tomorrowsGoal?: string | null;
  rating?: number | null;
  constructor(date?: string | null, mood?: string | null, affirmation?: string | null, achievement?: string | null,
    nonAchievement?: string | null, tomorrowsGoal?: string | null, rating?: number | null) {
    this.date = date;
    this.mood = mood;
    this.affirmation = affirmation;
    this.achievement = achievement;
    this.nonAchievement = nonAchievement;
    this.tomorrowsGoal = tomorrowsGoal;
    this.rating = rating;
  }
}
